docs/user-guide/spatial-hashing.ipynb

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
<p>Can we print the face indices and/or coordinates here to showcase what the output looks like? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I can do that, but this checkbox in the PR template conflicts with this request : " Clear the output of all cells before committing"

Reply:
Output is retained in https://github.com/UXARRAY/uxarray/pull/1169/commits/fc59591fae435817c73a24ecaf83be0c91ce0998

Reply:
The wording is pretty confusing, it's intended to mean that the notebook should be restarted and cleared before committing. 

I think our default template is due for an overhaul anyways, thanks for noting this.